### DISCUSSION TITLE
Introduce config option to hide keywords from being displayed

### DISCUSSION THREAD

**Discussion by salim-b** [quarto-dev/quarto-cli#10205] [Feature Requests]
### Description

If we set [`format.html.keywords`](https://quarto.org/docs/reference/formats/html.html#metadata), the HTML output produced by latest Quarto v1.5.53 **visually renders the keywords below the document/chapter title** besides including them as document metadata (by adding `<meta name="keywords" content="...">` in `<head>`):

![Screenshot 2024-07-02 at 22-41-22 2 Test chapter – The World of Referendums](https://github.com/quarto-dev/quarto-cli/assets/20040931/56e1ff50-f88f-4885-b22d-d154ae2de009)

It would be great if this could be made opt-out (or even opt-in as I guess displaying keywords to readers is rarely wanted) via a new boolean option like `format.html.display-keywords`. If set to `false`, only the metadata in `<head>` would be added.

**Comment by cscheid** [comment]
If we're going to do that, I think we need to do it more comprehensively than just for keywords, so it requires design.

In the meantime, you should be able to hide these with CSS if you need to. Does that workaround work for you?

**Reply by salim-b** [reply]
Ok, I see.

> In the meantime, you should be able to hide these with CSS if you need to. Does that workaround work for you?

Yes, thanks. I think `body main > header div.keywords` is the most precise selector we can use here.
